.github/workflows/run_tests_ci_workflow.yml

```
@@ -9,9 +9,18 @@ jobs:
     name: Run Tests
     runs-on: ubuntu-latest
     steps:
+      - name: Install clang-format
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y clang-format
       - name: Checkout repository
         uses: actions/checkout@v5
 
+      - name: Clang Format
+        run: clang-format --dry-run include/*.h source/*.c > /dev/null 2>&1
+        # When existing code conforms, re-add this:
+        # run: clang-format --dry-run -Werror include/*.h source/*.c
```

```suggestion
        run: clang-format --dry-run -Werror include/bitset.h source/bitset.c
        # Currently contains only the code that has been formatted to conform
        # When all existing code conforms, change to this:
        # run: clang-format --dry-run -Werror include/*.h source/*.c
```

Why not like this? That way we can add every file when we format it and make sure it stays that way, right?